src/functions/source_extract/tests.py

```diff
@@ -7,32 +8,61 @@
     'redhum': {
         'url': 'https://redhum.org/documento/123212',
         'response': {
-            "source_raw": "redhum",
-            "author_raw": "United Nations Commission on Human Rights",
-            "title": "Six Rapporteurs address Human Rights Commission under debate on civil and political rights",
-            "date": "2003-04-08",
-            "country": "Afghanistan",
-            "website": "redhum.org",
-            "url": "https://redhum.org/documento/123212",
+            'sourceRaw': 'redhum',
+            'authorsRaw': ['United Nations Commission on Human Rights'],
+            'title': (
+                'Six Rapporteurs address Human Rights Commission '
+                'under debate on civil and political rights'
+            ),
+            'date': '2003-04-08',
+            'country': 'Afghanistan',
+            'website': 'redhum.org',
+            'pdfUrls': [],
         },
     },
-    'reliefweb-working': {
+    'reliefweb-working-pdf': {
         'url': 'https://reliefweb.int/sites/reliefweb.int/files/resources/ReliefWeb%20Vision%20and%20Strategy.pdf',
         'response': {
-            'author_raw': None,
+            'authorsRaw': [],
             'country': None,
             'date': None,
-            'source_raw': 'reliefweb',
+            'sourceRaw': 'reliefweb',
             'title': 'ReliefWeb Vision and Strategy',
-            'url': 'https://reliefweb.int/sites/reliefweb.int/files/resources/ReliefWeb%20Vision%20and%20Strategy.pdf',
             'website': 'reliefweb.int',
+            'pdfUrls': [],
+        },
+    },
+    'reliefweb-working-url': {
+        'url': 'https://reliefweb.int/report/bangladesh/millions-bangladesh-impacted-one-worst-floodings-ever-seen',
+        'response': {
+            'authorsRaw': ['IFRC'],
+            'country': 'Bangladesh',
+            'date': '2022-06-28',
+            'sourceRaw': 'reliefweb',
+            'title': 'Millions in Bangladesh impacted by one of the worst floodings ever seen',
+            'website': 'reliefweb.int',
+            'pdfUrls': [],
+        },
+    },
+    'reliefweb-working-url-with-pdf': {
+        'url': 'https://reliefweb.int/report/south-sudan/south-sudan-nutrition-service-delivery-covid-19-context-frequently-asked-question-and-suggested-measures',  # noqa:E501
+        'response': {
+            'authorsRaw': ['Nutrition Cluster', 'UNICEF'],
+            'country': 'South Sudan',
+            'date': '2020-05-28',
+            'sourceRaw': 'reliefweb',
+            'title': 'South Sudan: Nutrition Service Delivery In COVID-19 Context - Frequently asked question and suggested measures',  # noqa:E501
+            'website': 'reliefweb.int',
+            'pdfUrls': [
+                'https://reliefweb.int//attachments/9fc61749-e380-460d-9809-bfcf3be77722/nutrition_covid_19_-_faq_and_recommendations_-_updated_29.05.2020_1.pdf'  # noqa:E501
+            ],
         },
```

Comment:
@AdityaKhatri @samshara 
Please look at this for the new API response structure.